# detectors中object3d_detector配置的顺序需要跟tracking_classes中的顺序一致
filter_type: "EKF"                           # The Kalman filter type: EKF = Extended Kalman Filter, UKF = Uncented Kalman Filter
std_limit: 0.6
cv_noise_params:                             # The noise for the constant velocity prediction model
    x: 1.4
    y: 1.4
    std_limit: 1.0                           # upper limit for the standard deviation of the estimated position 
detectors:                                   # Add detectors under this namespace
    car3d_detector:                       # Name of detector (used internally to identify them). Has to be unique.
        observation_model: "CARTESIAN3DYaw"       # Obeservation model: CARTESIAN or POLAR or CARTESIAN3D
        noise_params:                        # The noise for the cartesian or polar observation model
            x: 0.3
            y: 0.3
        matching_algorithm: "NN"             # The algorthim to match different detections. NN = Nearest Neighbour, NNJPDA = NN Joint Probability Data Association
        create_seq_size: 3                   # Minimum number of observations for new track creation
        create_seq_time: 1.0                 # Minimum interval between observations for new track creation
        prune_seq_size: 20                   # Minimum number of observations for prune track
    person3d_detector:                       # Name of detector (used internally to identify them). Has to be unique.
        observation_model: "CARTESIAN"     # Obeservation model: CARTESIAN or POLAR or CARTESIAN3D
        noise_params:                        # The noise for the cartesian or polar observation model
            x: 0.3
            y: 0.3
        matching_algorithm: "NN"             # The algorthim to match different detections. NN = Nearest Neighbour, NNJPDA = NN Joint Probability Data Association
        create_seq_size: 3                   # Minimum number of observations for new track creation
        create_seq_time: 1.0                 # Minimum interval between observations for new track creation
        prune_seq_size: 20                   # Minimum number of observations for prune track
tracking_classes: [car, person]